src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -173,24 +173,31 @@
 @snippet methodCallSampleCode(apiMethod)
     @if apiMethod.isSingularRequestMethod
         ${@apiMethod.apiVariableName}->{@apiMethod.name}(\
-            {@sampleMethodCallArgList(apiMethod.initCode.fieldSettings)})
+            {@sampleMethodCallArgList(apiMethod.initCode)})
     @else
         ${@apiMethod.apiVariableName}->{@apiMethod.name}()
     @end
 @end
 
-@private pageSizeMethodCallSampleCode(apiMethod)
-    @if apiMethod.hasRequiredParameters
-        ${@apiMethod.apiVariableName}->{@apiMethod.name}(\
-            {@sampleMethodCallArgList(apiMethod.initCode.fieldSettings)}, ['pageSize' => 5])
```

Comment:
I've changed the samples for PHP to not print `['pageSize' => 5]` as an optional parameter, mostly because it becomes difficult to actually insert an override in the gapic yaml for paging samples.

I think this is okay, as no other language is explicit about page sizes in the samples.